src/Database/Type/DateTimeType.php

```diff
@@ -64,6 +64,7 @@ class DateTimeType extends BaseType implements BatchCastingInterface
         'Y-m-d\TH:i',
         'Y-m-d\TH:i:s',
         'Y-m-d\TH:i:sP',
+        'Y-m-d',
```

Comment:
Move this to the front of the array.

Reply:
Since date only string would be a uncommon format for datetime, wouldn't it be better to keep it last?

Reply:
Ok may be it's not a good idea

Reply:
I gave up trying to optimize the list and just made it readable/grouped. We can leave it at the bottom though.

Reply:
Shouldn't this be in the DateType? DateTimeType is for date and time.